selfdrive/car/volkswagen/volkswagencan.py

```diff
@@ -16,27 +16,20 @@ def create_mqb_steering_control(packer, bus, apply_steer, idx, lkas_enabled):
   return packer.make_can_msg("HCA_01", bus, values, idx)
 
 def create_mqb_hud_control(packer, bus, enabled, steering_pressed, hud_alert, left_lane_visible, right_lane_visible,
-                           ldw_lane_warning_left, ldw_lane_warning_right, ldw_side_dlc_tlc, ldw_dlc, ldw_tlc,
-                           standstill, left_lane_depart, right_lane_depart):
+                           ldw_stock_values, left_lane_depart, right_lane_depart):
   # Lane color reference:
-  # 0 (LKAS disabled) - off 
-  # 1 (LKAS enabled, no lane detected) - dark gray 
-  # 2 (LKAS enabled, lane detected) - light gray on VW, green or white on Audi depending on year or virtual cockpit.  On a color MFD on a 2015 A3 TDI it is white, virtual cockpit on a 2018 A3 e-Tron its green. 
-  # 3 (LKAS enabled, lane departure detected) - white on VW, red on Audi 
-
+  # 0 (LKAS disabled) - off
+  # 1 (LKAS enabled, no lane detected) - dark gray
+  # 2 (LKAS enabled, lane detected) - light gray on VW, green or white on Audi depending on year or virtual cockpit.  On a color MFD on a 2015 A3 TDI it is white, virtual cockpit on a 2018 A3 e-Tron its green.
+  # 3 (LKAS enabled, lane departure detected) - white on VW, red on Audi
   values = {
     "LDW_Status_LED_gelb": 1 if enabled and steering_pressed else 0,
     "LDW_Status_LED_gruen": 1 if enabled and not steering_pressed else 0,
     "LDW_Lernmodus_links": 3 if left_lane_depart else 1 + left_lane_visible,
     "LDW_Lernmodus_rechts": 3 if right_lane_depart else 1 + right_lane_visible,
     "LDW_Texte": hud_alert,
-    "LDW_SW_Warnung_links": ldw_lane_warning_left,
-    "LDW_SW_Warnung_rechts": ldw_lane_warning_right,
-    "LDW_Seite_DLCTLC": ldw_side_dlc_tlc,
-    "LDW_DLC": ldw_dlc,
-    "LDW_TLC": ldw_tlc
   }
-  return packer.make_can_msg("LDW_02", bus, values)
+  return packer.make_can_msg("LDW_02", bus, {**values, **ldw_stock_values})
```

Comment:
I think something like this would be more readable:
```
values = ldw_stock_values.copy()
values.update({
  "key": val,
})
return packer.make_can_msg("LDW_02", bus, values)
```

Reply:
Counter-proposal:

```python
values = {
  "foo": 0,
  "bar": 1,
  **ldw_stock_values,
}
return packer.make_can_msg("LDW_02", bus, values)
```

Thoughts?

Reply:
I prefer the other way since you don't have to think about the behavior for duplicate keys.